ops/ccs-ops-misc/scripts/bfd-monthly-inventory

```diff
@@ -0,0 +1,288 @@
+#!/usr/local/bin/bash
+set -eo pipefail
+
+# global variables
+PROGNAME=${0##*/}
+INVENTORY_DIR=~/Desktop/monthly_inventory
+APPS_DIR=${APPS_DIR:-../../../apps}
+BUILD_APPS="${BUILD_APPS:-true}"
+BFD_SSH_USER=${BFD_SSH_USER:-} # must be set to ssh
+BFD_SSH_KEY_PATH=${BFD_SSH_KEY_PATH:-} # must be set to ssh
```

Comment:
this env variable name is not consistent with your bfd-ssh script.

Reply:
Good catch, the bfd-ssh script is still a wip, so I will update it there.